### DISCUSSION TITLE
possibility to add things on right bar menu

### DISCUSSION THREAD

**Discussion by spatiu** [chipzoller/hugo-clarity#165] [Ideas]
Hi, i don't know if already is possible with your theme, what i ask is for a easy way to add things like donation box (as exemple this [Donorbox )](https://donorbox.org/?utm_source=Blog) , i want to add a donation button similar with donorbox.org to right menu up before a short description about me, and i want to add other things like small images in part down of menu after  Tags.  In other way explain i want my menu in right can be in this way : 
1, ( Donation button)
2. (About me, where is and that button READ MORE)
3. (Recent posts)
4. (Category>Series>Tags + any more tags if i want to use )
5. ( Here i want add some image , similar with banner ads but manual add it there) , hope i am not limited at few images to add it , i want can add it one under after one , as exemple after this image, i add other images and so on .

Thanks , i appreciated your theme and your work :+1:  :)

**Comment by chipzoller** [comment]
We don't provide this level of flexibility out-of-the-box, but you're more than welcome to write your own HTML and CSS to achieve your result. That task, however, is out of scope for this theme.
